### PR TITLE
Storing gen ht in the event content

### PR DESCRIPTION
**Pull request by OlivierBondu**
tested on 
`DYJetsToLL_M-5to50_HT-100to200_TuneCUETP8M1_13TeV-madgraphMLM-pythia8`
and
`DYJetsToLL_M-5to50_HT-400to600_TuneCUETP8M1_13TeV-madgraphMLM-pythia8`

the variable is in the correct range

ready to be merged !

### CODE REVIEW

**Comment by OlivierBondu** [comment]
Well, of course validation fails:

```
/var/jenkins/workspace/Framework/CMSSW/CMSSW_7_4_15/src/cp3_llbb/Framework/bin/testFrameworkOutput.cc:118: FAILED:
  REQUIRE( test_tree->GetNbranches() == n_branches )
with expansion:
  152 == 151
```

how / where do I update the validation trees ?

**Comment by blinkseb** [comment]
You need to be on lxplus and simply run the script `generate_reference_trees.sh` in the `test`  folder

**Comment by OlivierBondu** [comment]
updated trees (I also renamed `setup.sh` into `first_setup.sh`, which is less confusing :) )

**Comment by OlivierBondu** [comment]
please test

**Comment by blinkseb** [comment]
Still not good

```
REQUIRE( test_tree->GetNbranches() == n_branches )
with expansion:
  155 == 154
```

Did you switch to your branch and rebuild before regenerating the trees?

**Comment by OlivierBondu** [comment]
I'm quite sure I did... let me try again

**Comment by OlivierBondu** [comment]
trying again adding the wget in the `generate_reference_trees.sh`

**Comment by blinkseb** [comment]
Thanks :)
